sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerConfig.java

```diff
@@ -172,4 +180,12 @@ public SpannerConfig withMaxCumulativeBackoff(ValueProvider<Duration> maxCumulat
   SpannerConfig withServiceFactory(ServiceFactory<Spanner, SpannerOptions> serviceFactory) {
     return toBuilder().setServiceFactory(serviceFactory).build();
   }
+
+  public SpannerConfig withRpcPriority(ValueProvider<RpcPriority> rpcPriority) {
+    return toBuilder().setRpcPriority(rpcPriority).build();
+  }
```

Comment:
If we don't make this a ValueProvider type, then we can also remove this whole method : )

Reply:
Done